### PR TITLE
added user profile page for logged in users

### PR DESCRIPTION
**Pull request by Brobin**
Added the user's profile as a pseudo-subreddit (`/r/me`). You can view the page by pressing `;` or by typing `me` in the subreddit prompt.

Currently only shows posts. I think some refactoring would have to take place in order to show comments as well.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -71,6 +71,8 @@ In subreddit mode you can browse through the top submissions on either the front
 :``►`` or ``l``: View comments for the selected submission
 :``/``: Open a prompt to switch subreddits
 :``f``: Open a prompt to search the current subreddit
+:``p``: Post a Submission to the current subreddit
+:``;``: View your Submissions (requires login)
```
Comment (michael-lazar) [inline README.rst]: I don't see the need for a new keybind, ``/r/me`` fits in really well already

Comment (Brobin) [inline README.rst]: Sounds good. That key might be useful for something down the road.

File: rtv/content.py [hunk 2]
```diff
@@ -309,6 +309,19 @@ def from_name(cls, reddit, name, loader, order='hot', search=None):
 
         return content
 
+    @classmethod
+    def from_reddittor(cls, reddit, username, loader, order='new'):
```
Comment (michael-lazar) [inline rtv/content.py]: Spelling? http://www.urbandictionary.com/define.php?term=redditor

Comment (Brobin) [inline rtv/content.py]: Wow, good catch :D

File: rtv/subreddit.py [hunk 3]
```diff
@@ -74,6 +76,16 @@ def prompt_subreddit(self):
         if name is not None:
             self.refresh_content(name=name)
 
+    def reddittor_profile(self):
+        if self.reddit.is_logged_in():
+            try:
+                self.content = SubredditContent.from_reddittor(
+                     self.reddit, self.username, self.loader)
```
Comment (michael-lazar) [inline rtv/subreddit.py]: Is there a way to get the username from the reddit session? We are adding a lot of new code just to store the username.

Comment (Brobin) [inline rtv/subreddit.py]: I tried getting the user by `self.reddit.get_me()` but I was getting an exception because it required OAuth.

Comment (Brobin) [inline rtv/subreddit.py]: Specific exception

    praw.errors.OAuthScopeRequired: `get_me` requires the OAuth2 scope `identity`

Comment (michael-lazar) [inline rtv/subreddit.py]: try
```python
reddit.user.name
```

Comment (Brobin) [inline rtv/subreddit.py]: On second thought, I like the second method better.

**Comment by Brobin** [comment]
Updated to use `reddit.user.get_submitted(sort=order)` instead of the `username` class attribute. I didn't know this existed in praw :D (I only saw the methods on the documentation)

**Pull request merged**